src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+
+    operate(action, number) {
+      this.result = action(number, this.result);
+
+      return this;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
```

Comment:
add/subtract/multiply/divide methods should have only one param and work with `this`
```suggestion
    add(value) {
      this.result += value;

      return this;
    },
```